BUILDING.md

```diff
@@ -28,7 +28,7 @@ On FreeBSD and OpenBSD, you may also need:
 * libexecinfo (FreeBSD and OpenBSD only)
 
 
-```text
+```sh
```

Comment:
@Fishrock123 Ah, previous commit merged with «sh». There is no difference though — you could use the file extension here, as `*.js` vs `*.javascript`.

Should I change it?